packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -45,6 +45,7 @@ const AppContents = observer(({ children }) => {
 
     React.useEffect(() => {
         if (is_logged_in && user_id) {
+            RudderStack.setAccountType(loginid.substring(0, 2));
```

Comment:
taken from https://github.com/binary-com/deriv-app/pull/9331/files since we don't know which PR will be merged sooner.